services/graph/pkg/service/v0/links.go

```diff
@@ -0,0 +1,146 @@
+package svc
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"path"
+	"strconv"
+
+	rpc "github.com/cs3org/go-cs3apis/cs3/rpc/v1beta1"
+	link "github.com/cs3org/go-cs3apis/cs3/sharing/link/v1beta1"
+	providerv1beta1 "github.com/cs3org/go-cs3apis/cs3/storage/provider/v1beta1"
+	"github.com/cs3org/reva/v2/pkg/utils"
+	"github.com/go-chi/render"
+	libregraph "github.com/owncloud/libre-graph-api-go"
+	"github.com/owncloud/ocis/v2/services/graph/pkg/linktype"
+	"github.com/owncloud/ocis/v2/services/graph/pkg/service/v0/errorcode"
+)
+
+func (g Graph) CreateLink(w http.ResponseWriter, r *http.Request) {
+	logger := g.logger.SubloggerWithRequestID(r.Context())
+	logger.Info().Msg("calling create link")
+	driveID, err := parseIDParam(r, "driveID")
```

Comment:
there is a new helper in my pr, we can change it afterwards...
https://github.com/owncloud/ocis/blob/9ca0810b8756fecc038f651af92e93d065701b38/services/graph/pkg/service/v0/utils.go#L35

Reply:
noted. I copied it to my pr.